server/src/internalClusterTest/java/org/opensearch/search/profile/aggregation/AggregationProfilerIT.java

```diff
@@ -150,6 +152,8 @@ public class AggregationProfilerIT extends OpenSearchIntegTestCase {
     private static final String TAG_FIELD = "tag";
     private static final String STRING_FIELD = "string_field";
     private final int numDocs = 5;
+    private static final String REASON_SEARCH_TOP_HITS = "search_top_hits";
+    private static final String REASON_AGGREGATION = "aggregation";
```

Comment:
Can we add a test with multiple aggregations as well and ensure the name of both the aggregations are in the output name ?

Reply:
Hi @sohami, I have added a new test.